man/systemd.link.xml

```diff
@@ -242,9 +242,11 @@
         <listitem>
           <para>An ordered, space-separated list of policies by which the interface name should be set.
           <varname>NamePolicy=</varname> may be disabled by specifying <option>net.ifnames=0</option> on the
-          kernel command line.  Each of the policies may fail, and the first successful one is used. The name
+          kernel command line. Each of the policies may fail, and the first successful one is used. The name
           is not set directly, but is exported to udev as the property <option>ID_NET_NAME</option>, which
-          is, by default, used by a udev rule to set <varname>NAME</varname>. The available policies are:
+          is, by default, used by a
+          <citerefentry><refentrytitle>udevadm</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
```

Comment:
```console
$ man 7 udevadm
No manual entry for udevadm in section 7
```

Reply:
Oops, this was supposed to be udev. Thanks for catching this.